pkg/volume/util/util.go

```diff
@@ -23,6 +23,7 @@ import (
 	"path"
 	"path/filepath"
 	"strings"
+	"syscall"
```

Comment:
I am not comfortable with `syscall` here, should be with platform dependent build.

Reply:
could you give me a hint about how to move "syscall" out here?
I found many other files also use "syscall".